### PR TITLE
Updating README with reference to spreequalle's PR

### PR DESCRIPTION
**Pull request by jonico**
* https://github.com/jonico/libzengithub/pull/34 solves essential tasks already
* follow-up items discussed in PR

/cc @spreequalle

### CODE REVIEW

**Comment by jonico** [comment]
@spreequalle: Upload to ARM64 successfully:

image

Can you kick the ARM32 agent again, it seems to be 😴

**Comment by jonico** [comment]
image

ARM32 coming back again ...

**Comment by spreequalle** [comment]
`2020-03-29 14:25:54Z: Job test (ARM) completed with result: Succeeded`
looks promising

**Comment by jonico** [comment]
Now ARM32 packages have been uploaded as well:

image

File: README.md [hunk 1]
```diff
@@ -209,6 +209,8 @@ conan upload ZenGitHub/1.0@jonico/stable --all -r=<your conan remote>
 
 ## :tada: GitHub SlowMerge / Octoplush Incentive :tada:
 
+### We have a [winner](https://github.com/jonico/libzengithub/pull/34) :tada:
```
Comment (jonico) [inline README.md:212]: test

**Pull request merged**